superset-frontend/src/setup/setupClient.ts

```diff
@@ -18,13 +18,18 @@
  */
 import { SupersetClient, logging, ClientConfig } from '@superset-ui/core';
 import parseCookie from 'src/utils/parseCookie';
+import getBootstrapData from 'src/utils/getBootstrapData';
+
+const bootstrapData = getBootstrapData();
 
 function getDefaultConfiguration(): ClientConfig {
   const csrfNode = document.querySelector<HTMLInputElement>('#csrf_token');
   const csrfToken = csrfNode?.value;
 
   // when using flask-jwt-extended csrf is set in cookies
-  const cookieCSRFToken = parseCookie().csrf_access_token || '';
+  const jwtAccessCsrfCookieName =
+    bootstrapData.common.conf.JWT_ACCESS_CSRF_COOKIE_NAME;
```

Comment:
Shouldn't we have a fallback here to `'csrf_access_token'`?

Reply:
by default `JWT_ACCESS_CSRF_COOKIE_NAME` is `csrf_access_token` and that default is set by flask-jwt-extended